app/grandchallenge/algorithms/models.py

```diff
@@ -407,6 +407,98 @@ def get_jobs_limit(self, user):
                 credits_left = user_credit.credits
             return max(credits_left, 0) // max(self.credits_per_job, 1)
 
+    @property
+    def usage_chart_statuses(self):
+        """What statuses should be included on the chart"""
+        return [Job.SUCCESS, Job.CANCELLED, Job.FAILURE]
+
+    @property
+    def usage_chart_status_choices(self):
+        """A map of int to string for Job.choice"""
+        return {k: v for k, v in Job.status.field.choices}
```

Comment:
Why not just `dict(ComponentJob.STATUS_CHOICES)`?

Reply:
ah probably a circular import issue

Reply:
Because the relationship between the field `status` and `ComponentJob.STATUS_CHOICES` is defined by `Job.status.field.choices`, but indeed we could do `dict(Job.status.field.choices)`.